### PR TITLE
Add `--version` to the `ChainSpecBuilder` command for `staging-chain-spec-builder`

### PR DESCRIPTION
**Pull request by bkontur**
## TODO

- [x] test/confirm that the release script is ok

### CODE REVIEW

File: .github/workflows/release-30_publish_release_draft.yml [hunk 1]
```diff
@@ -41,21 +42,17 @@ jobs:
...
-          if [[ ${{ matrix.binary }} =~ chain-spec-builder ]]; then
-            cargo build --locked --profile=production -p staging-${{ matrix.binary }} --bin ${{ matrix.binary }}
-            target/production/${{ matrix.binary }} -h
-          else
-            cargo build --locked --profile=production -p ${{ matrix.binary }}
-            target/production/${{ matrix.binary }} --version
-          fi
+          cargo build --locked --profile=production -p ${{ matrix.binary[0] }} --bin ${{ matrix.binary[1] }}
+          target/production/${{ matrix.binary }} --version
+          target/production/${{ matrix.binary }} -h
```
Comment (pepoviola) [inline .github/workflows/release-30_publish_release_draft.yml:49]: I think we need to update these lines to actually check the binary.

```suggestion
          target/production/${{ matrix.binary[1] }} --version
          target/production/${{ matrix.binary[1] }} -h
```

Comment (bkontur) [reply .github/workflows/release-30_publish_release_draft.yml]: @pepoviola ah, right, my bad, thank you :)

**Review by bkchr — APPROVED** [review]

File: .github/workflows/release-30_publish_release_draft.yml [hunk 2]
```diff
@@ -41,21 +42,17 @@ jobs:
...
-          if [[ ${{ matrix.binary }} =~ chain-spec-builder ]]; then
-            cargo build --locked --profile=production -p staging-${{ matrix.binary }} --bin ${{ matrix.binary }}
-            target/production/${{ matrix.binary }} -h
-          else
-            cargo build --locked --profile=production -p ${{ matrix.binary }}
-            target/production/${{ matrix.binary }} --version
-          fi
+          cargo build --locked --profile=production -p ${{ matrix.binary[0] }} --bin ${{ matrix.binary[1] }}
+          target/production/${{ matrix.binary[1] }} --version
+          target/production/${{ matrix.binary[1] }} -h
```
Comment (EgorPopelyaev) [inline .github/workflows/release-30_publish_release_draft.yml:49]:
```suggestion
```

just to check the version would be enough here, I think

**Review by EgorPopelyaev — APPROVED** [review]

**Comment by EgorPopelyaev** [comment]
@bkontur I tested the change looks good!👍🏻

**Pull request merged**